internal/global/trace_test.go

```diff
@@ -91,3 +91,22 @@ func TestTraceProviderDelegates(t *testing.T) {
 	gtp.Tracer("abc", trace.WithInstrumentationVersion("xyz"))
 	assert.True(t, called, "expected configured TraceProvider to be called")
 }
+
+func TestTraceProviderDelegates_SameInstance(t *testing.T) {
+	global.ResetForTest()
+
+	// Retrieve the placeholder TracerProvider.
+	gtp := otel.GetTracerProvider()
+
+	tracer := gtp.Tracer("abc", trace.WithInstrumentationVersion("xyz"))
+	assert.Same(t, tracer, gtp.Tracer("abc", trace.WithInstrumentationVersion("xyz")))
+	assert.Same(t, tracer, gtp.Tracer("abc", trace.WithInstrumentationVersion("xyz")))
```

Comment:
```suggestion
```

Reply:
Do we expect different results the second time? It sounds like I am missing something.

Reply:
Oh, doh, yeah nvm. Still need to get my first coffee :facepalm: